### ISSUE TITLE
windows defender says include what you use 0.6 contains a trojan

### ISSUE THREAD

**Issue by ThosRTanner** [include-what-you-use/include-what-you-use#337]
I hope this is  a false positive...0.5 is OK fortunately. I'd like some quick confirmation before bad things happen...

**Comment by kimgr** [comment]
This is for the Windows executable, right? I built it, and I don't know of anything suspicious on my corporate-antivirus-laden machine. I definitely didn't put anything bad there intentionally.

Are there any details around what it's triggering on?

**Comment by ThosRTanner** [comment]
Windows defender says 'trojan:win32/maltule.c!cl' - however the web page it directs you to seems broken.

**Comment by ThosRTanner** [comment]
malwarebytes says its ok though - it may well be windows defender producing a false +ve

**Comment by kimgr** [comment]
I searched for more details and saw a mention of VirusTotal, an online scanner. Someone has apparently already run it through, I found an analysis from July 5:
https://www.virustotal.com/en/url/8e85312eebb78e08db58dd04a6421a1a8a0319b9e7e3049850737105a5e55b6c/analysis/

It looks happy, so I'm pretty sure this is a false positive. Feel free to follow up with Microsoft, I'm closing this and hoping the next release doesn't hit the same issue.
